geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/protobufs/TupleProtoBuf.scala

```diff
@@ -50,6 +74,17 @@ trait TupleProtoBuf {
         multibandTileProtoBufCodec.decode(message.tiles.get))
   }
 
+  implicit def tupleSpatialKeyImageProtoBufCodec =
+    new ProtoBufCodec[(SpatialKey, Array[Byte]), ProtoTuple] {
+    def encode(tuple: (SpatialKey, Array[Byte])): ProtoTuple =
+      ProtoTuple(
+        spatialKey = Some(spatialKeyProtoBufCodec.encode(tuple._1)),
+        imageBytes = ByteString.copyFrom(tuple._2))
```

Comment:
`imageBytes` isn't the best name here, its already a stretch between PNG and GeoTiff case. Is `bytes` an option or is that a keyword?